src/templates/it/base.html

```diff
@@ -0,0 +1,170 @@
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} su Twitter{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} su GitHub{% endmacro %}
+{% macro onLinkedIn(authorName) %}{{authorName}} su LinkedIn{% endmacro %}
+{% macro website(authorName) %}{{authorName}} sito web{% endmacro %}
+
+{% macro edition(year) %}Edizione {{ year }}{% endmacro %}
+
+{% block footer_title %}Navigazione nel footer di pagina{% endblock %}
+{% block copyright %}© Web Almanac. Concesso in licenza da <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a>.{% endblock %}
+{% block accessibility_statement %}Dichiarazione di accessibilità{% endblock %}
+
+{% block featured_chapter %}Capitolo in primo piano{% endblock %}
+{% block featured_chapter_last_year %}Capitolo in primo piano<br/>dal Web Almanac {{ year|int -1 }}{% endblock %}
+
+{# Check if read_chapter already defined in child template as macros can't be overridden #}
+{% if not read_chapter %}
+{% macro read_chapter(chapter) %}Leggi il capitolo: <span class="featured-chapter-name">{{ chapter }}</span>{% endmacro %}
+{% endif %}
+{% if not read_last_years_chapter %}
+{% macro read_last_years_chapter(chapter) %}Leggi il capitolo {{ year|int-1 }} <span class="featured-chapter-name">{{ chapter }}</span>{% endmacro %}
+{% endif %}
+
+{% block contributors_description %}
+The Web Almanac has been made possible by the hard work of the web community. {{ self.contributors() }} people have volunteered countless hours in the planning, research, writing and production phases.
+Il Web Almanac è stato reso possibile grazie al duro lavoro della web community. {{self.contributors() }} persone si sono offerte volontarie per innumerevoli ore nelle fasi di pianificazione, ricerca, scrittura e produzione.
+{% endblock %}
+{% block contributors_link %}Vedi i contributori{% endblock %}
+
+{% block methodology_stat_1_title %}Siti web testati{% endblock %}
+{% block methodology_stat_2_title %}Dati trattati{% endblock %}
+{% block methodology_link %}Scopri la nostra metodologia{% endblock %}
+
+{% block introduction %}Introduzione{% endblock %}
+
+{% block foreword_title %}Prefazione{% endblock %}
+
+{% block coming_soon %}Prossimamente{% endblock %}
+
+{% block appendix %}Appendice{% endblock %}
+{% block appendices %}Appendici{% endblock %}
+
+{% block ebook_title %}Ebook{% endblock %}
+{% block ebook_download_short %}Ebook PDF ({{ ebook_size_in_mb }}MB){% endblock %}
+{% block ebook_download %}Download l'intera Web Almanac del {{ year }} in formato PDF ({{ ebook_size_in_mb }}MB){% endblock %}
+{% block ebook_download_note %}(Generato con <a href="https://www.princexml.com/">www.princexml.com</a>){% endblock %}
+
+{%
+  set localizedPartTitles = {
+    "I. Page Content": "I. Page Content",
+    "II. User Experience": "II. User Experience",
+    "III. Content Publishing": "III. Content Publishing",
+    "IV. Content Distribution": "IV. Content Distribution"
```

Comment:
Can we translate the second one?

```suggestion
    "I. Page Content": "I. Contenuto della Pagina",
    "II. User Experience": "II. L'Esperienza Utente",
    "III. Content Publishing": "III. Pubblicazione di Contenuti",
    "IV. Content Distribution": "IV. Distribuzione dei Contenuti"
```